PEAR.php

```diff
@@ -89,64 +89,64 @@ class PEAR
     /**
      * Whether to enable internal debug messages.
      *
-     * @var     bool
+     * @var bool
      * @access  private
      */
-    var $_debug = false;
+    public $_debug = false;
 
     /**
      * Default error mode for this object.
      *
-     * @var     int
+     * @var int
      * @access  private
      */
-    var $_default_error_mode = null;
+    public $_default_error_mode = null;
```

Comment:
public or private? Docblock disagrees with implementation, maybe better to just leave these as is